internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/BlobTransferUtil.java

```diff
@@ -32,10 +33,12 @@ public static BlobTransferManager<Void> getP2PBlobTransferManagerAndStart(
       String baseDir,
       ClientConfig clientConfig) {
     try {
+      AvroGenericStoreClientImpl storeClient =
```

Comment:
We should not casting type randomly anywhere, could you also fix this?

Reply:
Fixed in latest commit